src/Upgrade/Application/Bridge/IntegratorBridgeInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrade\Application\Bridge;
+
+use Upgrade\Application\Dto\StepsExecutionDto;
+
+interface IntegratorBridgeInterface
```

Comment:
It seems like an adapter and not bridge

Reply:
It seems like an adapter and not bridge.  - In his case we have to remain all Bridge dir to Adapter (previously we discussed that it is correct name)

Should it be part of Core? - To my mind no

Reply:
++

Reply:
Renamed to IntegratorAdapterInterface